From 9dc0072e8d5a47fb0b2ada50fb897cb2c740bb40 Mon Sep 17 00:00:00 2001
From: Thierry Reding <thierry.reding@avionic-design.de>
Date: Fri, 18 Jan 2013 13:22:01 +0100
Subject: [PATCH 4/7] man: Fix typo and use $() for make expressions

Due to the typo, none of the .xml files would end up in the release
tarball and cause make distcheck as well as builds from the tarball to
fail.

Using $() isn't strictly necessary but other variables and expressions
use that variant already so it makes the usage consistent.

Reviewed-by: David Herrmann <dh.herrmann@googlemail.com>
Signed-off-by: Thierry Reding <thierry.reding@avionic-design.de>
---
 man/Makefile.am |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index 25202e2..d25a293 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -17,7 +17,8 @@ MANPAGES_ALIASES = \
 	drm-ttm.7
 
 XML_FILES = \
-	${patsubst %.1,%.xml,${patsubst %.3,%.xml,${patsubst %.5,%.xml,${patsubs %.7,%.xml,$(MANPAGES)}}}}
+	$(patsubst %.1,%.xml,$(patsubst %.3,%.xml,$(patsubst %.5,%.xml,$(patsubst %.7,%.xml,$(MANPAGES)))))
+
 EXTRA_DIST = $(XML_FILES)
 CLEANFILES = $(MANPAGES) $(MANPAGES_ALIASES) .man_fixup
 man_MANS =
-- 
1.7.9.5

